modules/plugin/imagemosaic/src/test/java/org/geotools/gce/imagemosaic/ImageMosaicReaderTest.java

```diff
@@ -2491,7 +2495,7 @@ public void multipleDimensionsStackedSar() throws Exception {
 
         final String[] metadataNames = reader.getMetadataNames();
         assertNotNull(metadataNames);
-        assertEquals(15, metadataNames.length);
+        assertEquals(16, metadataNames.length);
```

Comment:
What happened here? I see it does not match any of the constants.

Reply:
It was a different isolated test already returning a different number of entries with respect to the others. I didn't add constants for all the possible metadataNames.length across the whole JUNIT Test class. Do you think it's better to put all the possible values into constants?